nuxeo-services/nuxeo-web-resources-core/src/main/java/org/nuxeo/ecm/web/resources/core/ResourceBundleDescriptor.java

```diff
@@ -99,6 +111,10 @@ public ResourceBundle merge(ResourceBundle other) {
             if (res != null) {
                 merged.addAll(res);
             }
+            if (merged.removeAll(Arrays.asList("", null))) {
+                log.error("Some resources references were null or blank while setting " + this.name
+                    + " and have been supressed. This probably happened because some <resource> tags were empty in the xml declaration. The correct form is <resource>resource name</resource>.");
+            }
```

Comment:
You the same block of code, you can put this in a private method.
Furthermore I'm not sure if the case is possible in merge method as all resources set seem to call setResources.